TriblerGUI/test/widgets/trustpage/js/test_data_processor.js

```diff
@@ -188,6 +189,44 @@ describe('data_processor.js', function () {
         });
     });
 
+    describe('addTrafficFunction', function () {
+       it('should return values for empy list of nodes', function () {
+          var data = getTestData();
+          var response = {
+              nodes: [],
+              edges: []
+          };
+
+          var result = processor.convertResponse(response, [
+              processor.mapNodes,
+              processor.mapEdges,
+              processor.addTrafficFunction
+          ]);
+
+          assert.equal(result.traffic_slope, config.node.circle.maxRadius - config.node.circle.minRadius);
+          assert.equal(result.traffic_base, config.node.circle.minRadius);
+       });
```

Comment:
These lines can be removed once you separate the config from the data processor.